src/NServiceBus.RabbitMQ/RabbitMqDequeueStrategy.cs

```diff
@@ -280,7 +279,26 @@ void Purge()
             }
         }
 
+        bool MovePoisonMessage(BasicDeliverEventArgs message, string queue)
+        {
+            var connection = connectionManager.GetPublishConnection();
+            var success = true;
+
+            using (var channel = connection.CreateModel())
+            {
+                try
+                {
+                    channel.BasicPublish("", queue, false, message.BasicProperties, message.Body);
```

Comment:
In 4.x, we seem to be passing `true` for `mandatory`. Is there a reason we're passing `false` here?